internal/search/job/jobutil/combinators.go

```diff
@@ -120,6 +124,12 @@ func (t *TimeoutJob) Name() string {
 	return "TimeoutJob"
 }
 
+func (t *TimeoutJob) Tags() []log.Field {
+	return []log.Field{
+		log.Int64("timeout", int64(t.timeout)),
```

Comment:
Can we format this as a human-readable string? `15s` is a bit more readable than `15000000000`
```suggestion
		log.Stringer("timeout", t.timeout),
```

Reply:
Thanks, changed